developer-docs-site/docs/nodes/ait/connect-to-testnet.md

```diff
@@ -227,13 +193,64 @@ You can check the details about node liveness definition [here](https://aptos.de
     
     You should expect the active value for your StakePool to keep increasing. It's updated at every epoch, so it will be every two hours.
 
+Go to the bottom of this page if you want to know how to Add Monitoring Components (terraform only).
 
 ## Leaving Validator Set
 
-A node can choose to leave validator set at anytime, or it would happen automatically when there's not sufficient stake on the validator account. To leave validator set, you can perform the following steps:
+A node can choose to leave validator set at anytime, or it would happen automatically when there's not sufficient stake on the validator account. Before you shutdown the node, you should also need to make sure you leave validator set first (will take effect in next epoch). To leave validator set, you can perform the following steps:
 
 1. Leave validator set (will take effect in next epoch)
 
     ```
     aptos node leave-validator-set --profile ait3-operator --pool-address <owner-address>
     ```
+## Shutdown Nodes for Incentivized Testnet
+
+If you have ensured that your node left the validator set, you can now take down the validator node and cleanup the resources used by the node.
+
+### Using source code
+
+- Stop your node.
+- Remove the data directory: `rm -rf <your-data-directory>`
```

Comment:
We should avoid telling people to use `rm -rf` forcing, is not preferred because you can really do some damage.

Reply:
what is the suggested language here?